### DISCUSSION TITLE
Delta hyperlinks not playing well with lazygit

### DISCUSSION THREAD

**Discussion by tg0h** [jesseduffield/lazygit#2250] [Q&A] [answered]
<img width="1449" alt="image" src="https://user-images.githubusercontent.com/7954426/200537702-926f3eec-c9d3-4e37-a19f-8ff69d128c6a.png">

I use delta as my git pager. When I enable hyperlinks in delta, my lazygit output looks funny. Has anyone had this problem before?

**Comment by Ryooooooga** [comment, marked as answer]
https://github.com/jesseduffield/gocui/pull/24
